src/app/workflow/workflow.component.html

```diff
@@ -244,12 +244,17 @@ <h3>
             ></app-info-tab>
           </mat-tab>
           <div *ngIf="entryType === EntryType.Notebook">
-            <mat-tab id="codeTab" label="Code">
-              <div class="mt-4 ml-4 mr-4">
-                <p>A user-friendly representation of the notebook will appear here, after it is implemented in SEAB-5117.</p>
-                <p>For a Jupyter notebook, it'll be the contents of the code/documentation cells, nicely-formatted.</p>
-                <p>For other kinds of notebooks, it might be something different.</p>
-              </div>
+            <mat-tab id="previewTab" label="Preview">
+              <ng-template matTabContent>
+                <div class="mt-4 ml-4 mr-4">
+                  <app-formatted-notebook
+                    *ngIf="workflow && selectedVersion && (extendedWorkflow$ | async)"
+                    [workflow]="workflow"
```

Comment:
IDE is giving me compiler errors for this and next line. We have this turned off at build time because there are too many of the errors, sigh, but we should try to avoid introducing new ones. Might be tricky here though.